wiki/People/The_Team/Nomination_Assessment_Team/Evaluations/en.md

```diff
@@ -10,32 +10,6 @@ If an assigned NAT member is unavailable or otherwise cannot do an assigned eval
 
 References to Discord in this article pertain to the BN Discord server, which has limited entry to the BN, NAT, GMT, and former members for organisation, discussion, and group socialisation.
 
-## Beatmap Nominator applications
```

Comment:
Would it make sense to just do a `Main page: /wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/` link instead of just outright removing all mention of apps on this page?